gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py

```diff
@@ -154,14 +165,22 @@ def AuthenticateOutOfBand(self, requestParameters):
 
             return authenticationService.authenticate(userId)
         else:
+            telemetry["result"] = "failed"
+            telemetry["reason"] = "invalid code"
+            if contact is not None:
+                telemetry["step"] = "code verification"
```

Comment:
isn't a step for both, registration and auth?

Reply:
Yes. https://github.com/sign-in-canada/Acceptance-Platform/blob/ebdb3eeb376670782dd4b84b5143b459028957ae/gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py#L170-L174

Reply:
in that case I think line 171 should site outside the if/else block

Reply:
It's the 2nd of 2 steps for registration, but the only step for authentication. No harm in including it for both however.